mteb/tasks/Audio/AudioPairClassification/eng/ESC50.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+import random
+import datasets
+import pandas as pd
+import numpy as np
+from mteb.abstasks.Audio.AbsTaskAudioPairClassification import (
+    AbsTaskAudioPairClassification,
+)
+
+random.seed(42)
+
+class ESC50PairClassification(AbsTaskAudioPairClassification):
+    metadata = TaskMetadata(
+        name="ESC50_PairClassification",
+        description="Environmental Sound Classification Dataset.",
+        reference="https://huggingface.co/datasets/ashraq/esc50",
+        dataset={
+            "path": "ashraq/esc50",
+            "revision": "e3e2a63ffff66b9a9735524551e3818e96af03ee",
+        },
+        type="AudioPairClassification",
+        category="a2a",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="max_ap",
+        date=("2023-01-07", "2023-01-07"),
+        domains=[
+            "Encyclopaedic"
+        ], 
```

Comment:
```suggestion
        domains=["Environmental sounds"], 
```
What type of environmental sounds? Maybe we can do better?